Pipfile

```diff
@@ -4,12 +4,12 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-flask = "*"
-structlog = "*"
-
-[dev-packages]
```

Comment:
Are we ok with this removing dev-packages?

Reply:
I'm happy to have with or without, but just wondering if there's a reason to have them?
I'm guessing the idea is to not have the tests/lints in the actual build, but how much space do they actually take up?
@emilioward Thoughts?

Reply:
I'm also guessing the idea is to run `pipenv .... --dev` ? So we would need an extra command in the Makefile

Reply:
Yer, I think the idea is to not include dev dependencies in the proper builld.  
Speaking of the makefile I was thinking we may want a `setup` command that runs  
`pipenv clean`  
`pipenv install --dev`  
`pre-commit install`

could maybe do that stuff in another ticket though